.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/devcontainers/python:3.11-bullseye
+
+RUN apt-get update && \
+    apt-get install -y python3-venv git curl && \
+    rm -rf /var/lib/apt/lists/*
+
+# Copy the devcontainer folder first
+COPY . /workspaces/py-launch-blueprint-main/.devcontainer
+
+# Then copy the rest of the repo (excluding .devcontainer)
+COPY .. /workspaces/py-launch-blueprint-main
+
+WORKDIR /workspaces/py-launch-blueprint-main
+
+# Make sure the setup script is executable
+RUN chmod +x /workspaces/py-launch-blueprint-main/.devcontainer/setup.sh
+
+# Install 'just' command runner
+RUN curl -sSL https://just.systems/install.sh | bash -s -- --to /usr/local/bin
```

Comment:
Make this PR depend on #256 and update your just install command based on it.

- https://github.com/smorin/py-launch-blueprint/pull/256